MaterialSkin/Search.pm

```diff
@@ -25,7 +25,7 @@ use Slim::Utils::Log;
 use Slim::Utils::Prefs;
 
 sub advancedSearch {
-	my $params  = shift;
+	my ($client, $params) = @_;
```

Comment:
Why? `client` is not used.

Reply:
Your implementation does not use it, but LMS' does. By keeping the function's signature identical, we can simplify the calling side.

Feel free to replace `$client` with `undef` if you don't like to have an unused variable defined.

Reply:
Fair enough :)